### PR TITLE
Updates to storing source code

### PR DESCRIPTION
**Pull request by SteveEvansGDS**
Minor style changes. Can someone check for technical accuracy please?

### CODE REVIEW

File: source/standards/source-code.html.md.erb [hunk 1]
```diff
@@ -11,25 +11,29 @@ At GDS, we follow the principles set out in the Service Manual for managing the
...
+Put new repositories for GDS services in the [alphagov](https://github.com/alphagov/) organisation on GitHub.
 
-You can use your personal GitHub account to access alphagov. Ask your tech lead or technical architect to invite you.
+You can use your personal GitHub account to access alphagov. Ask your tech lead or technical architect to invite you. GDS will revoke your access to alphagov when you leave GDS.
 
 To secure your Github repository, make sure you:
 
 - configure two-factor authentication for your account
-- ensure Git repositories are backed up to another location (this should be a team responsibility)
+- backup Git repositories to another location (this should be a team responsibility)
```
Comment (galund) [inline source/standards/source-code.html.md.erb]: I don't know any teams that do this. Each repo is on every developer's machine, so it's mostly unnecessary?

File: source/standards/source-code.html.md.erb [hunk 2]
```diff
@@ -11,25 +11,29 @@ At GDS, we follow the principles set out in the Service Manual for managing the
...
 - have considered encrypting your repository contents
 - consider [protecting your master branch](https://help.github.com/articles/about-protected-branches/) to prevent changes being committed without a suitable review
 
 ### How to to retire applications
 
 If an application is no longer used in production, you should [archive](https://help.github.com/articles/archiving-repositories/) its repository.
 
-Update the application's README to detail why the repository has been archived, and link to a new location if the application has been superseded.
+Update the application's README to explain why the repository has been archived, and link to a new location if the application has been replaced by a newer application.
```
Comment (galund) [inline source/standards/source-code.html.md.erb]: FWIW I think "superseded" is less wordy and is perfectly clear :-)

File: source/standards/source-code.html.md.erb (hunk 1 above)
Comment (SteveEvansGDS) [reply source/standards/source-code.html.md.erb]: So is it something we should be doing? Or is the repo being on local machines enough?

File: source/standards/source-code.html.md.erb (hunk 2 above)
Comment (SteveEvansGDS) [reply source/standards/source-code.html.md.erb]: I've changed it back

File: source/standards/source-code.html.md.erb (hunk 1 above)
Comment (rhowe-gds) [reply source/standards/source-code.html.md.erb]: GOV.UK do this with a gitlab instance, I believe

Comment (SteveEvansGDS) [reply source/standards/source-code.html.md.erb]: Ok, I'm going to change this to an optional thing to do rather than a must. Thanks both.

**Pull request merged**